apps/jobs/models.py

```diff
@@ -146,6 +147,22 @@ def save(self, *args, **kwargs):
                     status=Submission.FAILED,
                     submitted_at__gte=datetime.date.today()).count()
 
+                submissions_in_month_count = Submission.objects.filter(
+                    challenge_phase__challenge=self.challenge_phase.challenge,
+                    participant_team=self.participant_team,
+                    challenge_phase=self.challenge_phase,
+                    submitted_at__gte=timezone.now().replace(day=1, hour=0, minute=0, second=0, microsecond=0)).count()
+
+                failed_count_in_month = Submission.objects.filter(
+                    challenge_phase=self.challenge_phase,
+                    participant_team=self.participant_team,
+                    status=Submission.FAILED,
+                    submitted_at__gte=timezone.now().replace(day=1, hour=0, minute=0, second=0, microsecond=0)).count()
+                if ((submissions_in_month_count + 1 - failed_count_in_month > self.challenge_phase.max_submissions_per_month) or
```

Comment:
Can you please tell me why you are doing +1 here?

Reply:
This will check if User's Current Submission [(`Successful Submission Made` + 1)<sup>th</sup>] is less than `max_submissions_per_month`.